lib/house.rb

```diff
@@ -0,0 +1,36 @@
+class House
+
+  attr_reader :price, :address, :rooms
+
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+  end
+
+  def add_room(room)
+    @rooms << room
+  end
+
+  def above_market_average?
+    house_price = @price.chars.map { |char| char.to_i }
```

Comment:
great job getting the house_price in integers - since you use it multiple times though, it really deserves to be a separate method (see my note on your attr_readers)